.github/workflows/continuous-deployment.yml

```diff
@@ -0,0 +1,43 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Deployment"
+
+on:
+  push:
+    tags:
+      - "**"
+
+jobs:
+  release:
+    name: "Release"
+
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+
+      - name: "Determine tag"
+        id: "determine-tag"
+        run: "echo \"::set-output name=tag::${GITHUB_REF#refs/tags/}\""
+
+      - name: "Create release"
+        id: "create-release"
+        uses: "actions/create-release@v1.0.0"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        with:
+          draft: false
+          prerelease: false
+          release_name: "${{ steps.determine-tag.outputs.tag }}"
+          tag_name: "${{ steps.determine-tag.outputs.tag }}"
+
+      - name: "Upload cs2pr"
+        uses: "actions/upload-release-asset@v1"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+        with:
+          asset_content_type: "text/plain"
+          asset_name: "cs2pr"
+          asset_path: "cs2pr"
+          upload_url: "${{ steps.create-release.outputs.upload_url }}"
```

Comment:
For reference, see https://github.com/ergebnis/composer-normalize/blob/435ba576a79bb00ae16105ef674d72430612bd7d/.github/workflows/continuous-deployment.yml - it's almost the same workflow.